marketplace_builder/views/pages/api-reference/graphql/graphql-glossary.liquid

```diff
@@ -0,0 +1,189 @@
+
+---
+converter: markdown
+metadata:
+  title: GraphQL Glossary
```

Comment:
Description is missing on purpose? :)

Reply:
Absolutely not. :) Thank you for noticing, I added a description now.